### PR TITLE
Adding Disaggregated Coordinator Configuration Documentation

### PR DESCRIPTION
**Pull request by swapsmagic**
Test Plan - Verified UI by building docs locally.
```
== NO RELEASE NOTE ==
```

### CODE REVIEW

File: presto-docs/src/main/sphinx/installation/deployment.rst [hunk 1]
```diff
@@ -143,6 +143,50 @@ will function as both a coordinator and worker, use this configuration:
     discovery-server.enabled=true
     discovery.uri=http://example.net:8080
 
+For disaggregated coordinator setup, you can use below minimal configuration:
```
Comment (aweisberg) [inline presto-docs/src/main/sphinx/installation/deployment.rst:146]: How many resource managers and coordinators can I have? How does this functionality work? I think there isn't quite enough information here from someone to deploy and operate the feature and know how/why.

**Review by aweisberg — CHANGES_REQUESTED** [review]
I think more information to alllow people to understand the feature and deploy it based on the documentation would make sense.

File: presto-docs/src/main/sphinx/installation/deployment.rst (hunk 1 above)
Comment (swapsmagic) [reply presto-docs/src/main/sphinx/installation/deployment.rst:146]: Added some details to help understand the functionality.

Comment (swapsmagic) [reply presto-docs/src/main/sphinx/installation/deployment.rst]: Seems the push didn't worked last time, updated now for real.

**Review by aweisberg — APPROVED** [review]
LGTM! Thank you for adding the additional context.

**Comment by aweisberg** [comment]
@tdcmeehan @rschlussel  @highker can one of you review and merge this?

**Pull request merged**